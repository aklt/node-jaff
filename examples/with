#begin coffee

value = 'Hello world!'
indent = (lines) ->
  lines.map (line) ->
    '    ' + line

#end

with...do...end

#with lines
Programs say
{value}
#do
#  filter !indent
#    filter sed -e 's/Hello/Greetings/'
{JSON.stringify(lines[1])}
#    end
#  end
#end
