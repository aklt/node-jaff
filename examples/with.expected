
with...do...end

    "Greetings world!\n"
